xil_carma_cloud/cdasim_config/.env

```diff
@@ -1,5 +1,6 @@
-DOCKER_TAG=develop
-DOCKER_ORG=usdotfhwastoldev
+DOCKER_TAG=carma-system-4.5.0
```

Comment:
The docker tag is develop for carma-cloud and cdasim and carma system for everything else. We may want to think about have we set tags in this case. What I have for my PR seems to currently work as intended. Not sure what this is attempting to fix.

Reply:
didn't realize the images were hardcoded to 4.5.0, I put this change after a glance of develop on some images, so yeah this is unnecessary